doc/source/templates/04_finetuning_llms_with_deepspeed/cluster_env.yaml

```diff
@@ -1,31 +1,33 @@
 # See https://hub.docker.com/r/anyscale/ray for full list of
 # available Ray, Python, and CUDA versions.
-base_image: anyscale/ray:2.5.1-py39-cu116
+base_image: {{ env["RAY_IMAGE_ML_NIGHTLY_GPU"] | default("anyscale/ray:2.6.1-py39-cu117") }}
```

Comment:
This is also not usable by the customer -- should this be a private yaml file under `release/` rather than in the template working dir?

Reply:
yeah these are the stuff that I was trying to get the releas to work. I'll revert these.